apps/mull-ui/src/app/pages/create-event/create-event.tsx

```diff
@@ -39,15 +53,18 @@ const CreateEventPage = ({ history }) => {
   const toastId = useRef(null);
   // GraphQL mutation hook to create events
   const [createEvent] = useMutation(CREATE_EVENT);
+  const [uploadFile] = useMutation(UPLOAD_PHOTO);
   // Image of Event
-  const [imageFile, setImageFile] = useState(null);
-
+  const [imageURLFile, setImageURLFile] = useState(null);
+  const [file, setFile] = useState<File>(null);
```

Comment:
We could add a line of documentation explaining what each hook is doing, from what i understand, the need for 2 hooks is so that we can keep the image preview on the page. So adding an extra line would help other dev's down the road who are working on this component

Reply:
Why do we need two hooks

Reply:
Oh yeah, added a new hook because if I set the state for the same hook, then the preview of the image becomes a question mark image.

Reply:
Yeah, I'd really like a quick comment to explaining this, since it might be confusing

Reply:
Agreed

Reply:
Saw you added comments, resolving this